api/api.go

```diff
@@ -128,6 +128,31 @@ func Submit(url string, iter *Iteration) (*Submission, error) {
 	return ps.Submission, nil
 }
 
+// List available problems for a language
+func List(language, host string) ([]string, error) {
```

Comment:
I like the way this function takes separate arguments as opposed to the composed url. Even providing host seems a bit off but until we have a Client struct for the api package this is fine. I think we should follow this same pattern for the other functions in this file. Any thoughts on this?